ui/app/components/ttl-picker2.js

```diff
@@ -0,0 +1,112 @@
+/**
+ * @module TtlPicker2
+ * TtlPicker2 components are used to enable and select TTL
```

Comment:
we should probably leave a comment here or outside of the JSDocs to explain when to use this TtlPicker vs. the old one, and drop a note in the old one stating our deprecation plan (if there is one 😅)